File: samples/SimpleFullStack/DotnetCoreApi/DataAccess/Entities/ProductAttribute.cs
namespace copoilot_sample.DataAccess.Entities
{
    public class ProductAttribute
    {
        public int AttributeID { get; set; }
        public int ProductID { get; set; }
        public required string AttributeName { get; set; }
        public required string AttributeValue { get; set; }

        // Navigation property
        public Product? Product { get; set; }
    }
}

File: sample-code/SimpleFullStack/Web/src/types/ProductAttribute.ts
export interface ProductAttribute {
	attributeID: number;
	productID: number;
	attributeName: string;
	attributeValue: string;
}

export interface AddProductAttribute {
	productID: number;
	attributeName: string;
	attributeValue: string;
}

export interface UpdateProductAttribute {
	attributeName: string;
	attributeValue: string;
}
